vite.config.ts

```diff
@@ -11,10 +11,10 @@ export default defineConfig({
 		// in-source testing
 		includeSource: ['src/**/*.{js,ts,svelte}'],
 		// Add @testing-library/jest-dom matchers & mocks of SvelteKit modules
-		setupFiles: ['./scripts/setupTest.ts'],
+		setupFiles: ['src/tests/setup.ts'],
```

Comment:
Does this not work if you add it to the existing setup file that exists?

Reply:
oh wow, I don't know how I missed that this files existed already, sorry for that. will refactor!

Reply:
fixed now, I though `./scripts/setupTest.ts` was some generic example path and not actually something that existed 🤦